.github/actions/aws-ci/action.yml

```diff
@@ -35,9 +41,10 @@ runs:
         AWS_ACCESS_KEY_ID: ${{ inputs.aws-access-key-id }}
         AWS_SECRET_ACCESS_KEY: ${{ inputs.aws-secret-access-key }}
         AWS_ACCOUNT_TYPE: ${{ inputs.aws-account-type }}
+        CLOUDBEAT_CONFIG: ${{ inputs.cloudbeat-config }}
```

Comment:
Can't we generate `cloudbeat-config` based on the `python-keyword-expression`?

I mean, the default is aws. I could just have `deploy/aws/cloudbeat-${{ inputs.pytest-keyword-expression }}.yml`. This decreases on config. We just need to align the pytest key to the config file name

Reply:
I so see further down that the key and file have different names. I'm also ok with you just ignoring the suggestion.

I just thought that one setting only could make it easier to use

Reply:
Resolved by having a separate action for asset inventory.